modules/utils.py

```diff
@@ -902,3 +787,37 @@ def check_version(cur_version: str, log) -> None:
                 "warning",
                 "Your version of AutoPWN Suite is outdated. Update is advised.",
             )
+
+def GetShodanVulns(host, shodan_api_key, log):
+    api = shodan.Shodan(shodan_api_key)
+    try:
+        host_info = api.host(host)
+        vulns = host_info.get("vulns", [])
+        log.logger("INFO", f"Found {len(vulns)} vulnerabilities for {host} from Shodan.")
+        open_ports = [service['port'] for service in host_info.get('data', [])]
+        formatted_vulns = []
+        for vuln in vulns:
+            formatted_vulns.append({
+                'title': vuln,
+                'CVEs': [vuln],
+                'severity_score': 0.0,
+                'exploitability': 0.0
+            })
+        return formatted_vulns, open_ports
+    except shodan.APIError as e:
+        log.logger("ERROR", f"Error fetching Shodan vulnerabilities: {e}")
+        return []
+
+def GetZoomEyeVulns(host, zoomeye_api_key, log):
+    # Initialize ZoomEye API client
+    api = ZoomEye(api_key=zoomeye_api_key)
+    
+    try:
+        # Perform the search
+        results = api.dork_search(f"ip:{host}", page=1)
+        vulns = results.get('matches', [])
+        log.logger("INFO", f"Found {len(vulns)} vulnerabilities for {host} from ZoomEye.")
+        return vulns
+    except Exception as e:
+        log.logger("ERROR", f"Error fetching ZoomEye vulnerabilities: {e}")
+        return []
```

Comment:
It might be better for program structre to move these 2 functions to respective modules.

Reply:
looks good honestly :)